soc/xtensa/nxp_adsp/rt6xx/Kconfig.defconfig.series

```diff
@@ -42,7 +42,10 @@ config LOG
 	default n
 
 config NXP_ADSP_SIGN_IMAGE
-	bool
+	bool # FIXME: Defined extra.
 	default n
 
+config NXP_INPUTMUX
```

Comment:
See comment above about `default y` in the `nxp_inputmux/Kconfig` file- if `default y` is present there then these lines won't be needed.